tests/testthat/test-shinytest2-init.R

```diff
@@ -56,3 +56,57 @@ testthat::test_that("e2e: init creates UI containing specified title, favicon, h
   )
   app$stop()
 })
+
+testthat::test_that("e2e: teal app initializes with Show R Code modal", {
+  app <- TealAppDriver$new(
+    data = simple_teal_data(),
+    modules = example_module(label = "Example Module")
+  )
+  app$wait_for_idle(timeout = default_idle_timeout)
+
+  # Check if button exists.
+  button_selector <- "#teal-main_ui-root-example_module-module-rcode-button"
```

Comment:
```suggestion
  button_selector <- sprintf("#%s-rcode-button", app$active_module_ns())
```

We can use the `active_module_ns` method to get the active module namespace.

Reply:
Same can be used in the code below too.

Reply:
whoah, thanks my man

Reply:
a311fb749f7f999311469574b62e4f250c844ad0